pyproject.toml

```diff
@@ -26,6 +26,8 @@ exclude = [
 ]
 
 [project.optional-dependencies]
+cudf = ["cudf>=23.08.00"]
```

Comment:
whilst we're stuck testing manually on kaggle, i think this'll have to be the default version that gets loaded on kaggle